tofu/geom/_core.py

```diff
@@ -5815,7 +5817,7 @@ def check_ff(self, ff, t=None, ani=None):
         nt = len(t) if hasattr(t, '__iter__') else 1
 
         # .. Test anisotropic case .......................................
-        if ani is None:
+        if ani is False:
```

Comment:
I don't get it, if ani is False it means the user wants to force it to False, we should not change it.
We should only change it if None (means the user did not specify it).